repo-scripts/api-documenter/src/documenters/MarkdownDocumenterHelpers.ts

```diff
@@ -118,14 +119,14 @@ export function getFilenameForApiItem(
 }
 
 // TODO: handle method overloads and namespace?
```

Comment:
Can this TODO be removed now?

Reply:
Thanks. I removed the mention of method overloads. Regarding namespaces, I believe thats something that will need a separate solution.